recipes/supervised_fine_tuning/torchtune/utils.py

```diff
@@ -0,0 +1,222 @@
+import os
+import re
+from typing import List
+
+MODELS = {
```

Comment:
it's a bit unfortunate that we have a whitelist of models here but I assume you have to. Can you add some documentation of what this is and why we need it and how you might add a new model that we didn't think of?